public/js/actions/chartsActions.js

```diff
@@ -0,0 +1,23 @@
+export const updateComposerVsIncopy = ({ chartData, startDate, endDate }) => ({
+    type: 'UPDATE_COMPOSER_VS_INCOPY',
+    chartData,
+    startDate,
+    endDate
+});
+//remeber to pass error.message to these functions
+export const getComposerVsIncopyFailed = (error) => ({
```

Comment:
Looks like here and on the rest of the actions you have an extra set of brackets around it 
They should be
```js
export const actionName = (args) => {
  ...
};
```